flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1694,8 +1706,8 @@ def test_attach_existing_nonmanifest_unknown(self):
         whether to wake up or not. That is, we're using out-of-date cached
         volume information.
 
-        In this case ``BlockDeviceDeployer.calculate_changes`` returns a
-        ``ActionNeeded`` in order to wake up the convergence loop.
+        In this case ``BlockDeviceDeployer.calculate_changes`` returns some
```

Comment:
So now it'll return a create? Kinda icky, but yeah semantically that'll work so I guess that's OK.